app/decorators/profile_decorator.rb

```diff
@@ -21,4 +21,8 @@ def country_name
   def mail_subscription
     profile.subscribed? ? "Subscribed" : "Not Subscribed"
   end
+
+  def total_circuits
+    profile.projects.count.zero? ? "No circuits" : profile.projects.count
```

Comment:
```suggestion
    profile.projects.exists? ?  profile.projects.count : "No circuits"
```

Reply:
Done. Out of curiosity, does this approach helps with performance too?

Reply:
Yes